### PR TITLE
Rename change log generator to release notes generator

### PR DESCRIPTION
**Pull request by nkolev92**
I renamed it, simplified it and added a readme to make the running a single step thing. 

cc @jebriede

### CODE REVIEW

File: ReleaseNotesGenerator/ReleaseNotesGenerator/NuGet-6.4.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+---
```
Comment (jebriede) [inline ReleaseNotesGenerator/ReleaseNotesGenerator/NuGet-6.4.md]: Was this included by accident? This is the 6.4 output. 😊

**Review by jebriede — APPROVED** [review]
Approved with suggestion, looks like the output file was accidentally included in the PR

File: ReleaseNotesGenerator/ReleaseNotesGenerator/NuGet-6.4.md (hunk 1 above)
Comment (nkolev92) [reply ReleaseNotesGenerator/ReleaseNotesGenerator/NuGet-6.4.md]: yep, deleted it :D

**Comment by zivkan** [comment]
@nkolev92 I wanted to give feedback 😭 

Should namespaces in all cs files be changed, so that when new files are added with the default namespace, they match?

**Comment by nkolev92** [comment]
> @nkolev92 I wanted to give feedback 😭
> 
> Should namespaces in all cs files be changed, so that when new files are added with the default namespace, they match?

I'll fix that. 

I was trying to get it done quickly so that JP can use the updated tool.

**Comment by nkolev92** [comment]
https://github.com/NuGet/Entropy/pull/96. 

That PR will wait for feedback for longer :)

**Pull request merged**